detections/application/splunk_command_and_scripting_interpreter_risky_spl_mltk.yml

```diff
@@ -0,0 +1,74 @@
+name: Splunk Command and Scripting Interpreter Risky SPL MLTK
+id: 19d0146c-2eae-4e53-8d39-1198a78fa9ca
+version: 1
+date: '2022-05-27'
+author: Abhinav Mishra, Kumar Sharad and Xiao Lin, Splunk
+type: Anomaly
+datamodel:
+- Splunk_Audit
+description: 'This detection utilizes machine learning model named "risky_command_abuse" trained from "Splunk Command 
+  and Scripting Interpreter Risky SPL MLTK Baseline". It should be scheduled to run hourly to detect whether a user 
+  has run searches containing risky SPL from this list 
+  https://docs.splunk.com/Documentation/Splunk/latest/Security/SPLsafeguards#Commands_that_trigger_the_warninga
+  with abnormally long running time in the past one hour, comparing with his/her past seven days history. This search 
+  uses the trained baseline to infer whether a search is an outlier (isOutlier ~= 1.0) or not (isOutlier~= 0.0)'
+search: '| tstats values(Search_Activity.user) AS user, sum(Search_Activity.total_run_time) AS run_time,
+  values(Search_Activity.search) as searches, count 
```

Comment:
Same comments that apply to the baseline apply here.

Reply:
done